crates/uv/tests/pip_install_scenarios.rs

```diff
@@ -4906,12 +4906,12 @@ fn transitive_yanked_and_unyanked_dependency_opt_in() {
 
     ----- stderr -----
     Resolved 3 packages in [TIME]
+    warning: `package-c==2.0.0` is yanked (reason: "Yanked for testing").
```

Comment:
Whoops, didn't we intentionally display this last?

Reply:
Is it significant?

Reply:
I think so, the installed list can be pretty long and it's not helpful if it's buried. We flagged this once already and moved it to the bottom.

Reply:
So much harder to show them at the end lol

Reply:
I know sorry 😬